index.html

```diff
@@ -16,30 +18,30 @@
                     <img src="./assets/ic_menu.svg" alt="Burger menu icon">
                 </button>
             </nav>
-            <nav id='menu-container'  class='full-width' aria-controls="open-menu-btn">
+            <nav id='menu-container' class='full-width' aria-controls="open-menu-btn">
                 <div class="top-image full-width"></div>
                 <button id='close-menu-btn' class="menu-btn">
                     <img src="./assets/Union.svg" alt="Cancel icon">
                 </button>
                 <ul>
                     <li>
-                        <a href="#introduction-m" class="poppins-font-regular">Hello</a>
+                        <a class="poppins-font-regular mobile-item" href="#introduction-m">Hello</a>
                     </li>
                     <li>
-                        <a href="#works-m" class="poppins-font-regular">Portfolio</a>
+                        <a class="poppins-font-regular mobile-item" href="#works-m">Portfolio</a>
                     </li>
                     <li>
-                        <a href="#about-m" class="poppins-font-regular">About</a>
+                        <a class="poppins-font-regular mobile-item" href="#about-m">About</a>
                     </li>
                     <li>
-                        <a href="#form-m" class="poppins-font-regular">Contact</a>
+                        <a class="poppins-font-regular mobile-item" href="#form-m">Contact</a>
                     </li>
                 </ul>
                 <div class="bottom-image full-width"></div>
             </nav>
         </section>
         <section class="desktop-nav">
-            <nav id='menu-container'  class='full-width' aria-controls="open-menu-btn">
+            <nav id='menu-container' class='full-width' aria-controls="open-menu-btn">
```

Comment:
- Good job done so far 👍 , but there are some little differences between your design and the given design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- Please increase the spacing between the links as seen in the Figma design 👍 👍 
<img width="213" alt="ererer" src="https://user-images.githubusercontent.com/73607512/208689575-42783232-99d0-43f5-8cba-bda5d0842a5e.PNG">

Figma design:
<img width="118" alt="bwrbrb" src="https://user-images.githubusercontent.com/73607512/208689602-5462f6eb-075f-4c07-9ac1-3960a379d4fc.PNG">